README.md

```diff
@@ -157,12 +157,12 @@ A more complete overview of our stack is available in
 
 ## Vulnerability disclosure
 
-Forem is the open source software which powers [DEV](https://dev.to).
+Forem is the open-source software which powers [DEV](https://dev.to).
```

Comment:
We have [more instances](https://github.com/forem/forem/search?q=open+source) of `open source` as [compared to](https://github.com/forem/forem/search?q=open-source) `open-source`, and I think we prefer the non-hyphenated version. If you'd like to fix the ones that _do_ have hyphens to be consistent, that would be awesome! 👍🏽 
```suggestion
Forem is the open source software which powers [DEV](https://dev.to).
```